lib/bolt/outputter/human.rb

```diff
@@ -78,7 +78,13 @@ def print_summary(results, elapsed_time)
       end
 
       def print_plan(result)
-        @stream.puts result
+        if result_hash = unwrap_execution_result(result)
+          unless result_hash.empty?
+            @stream.puts(::JSON.pretty_generate(result_hash))
+          end
+        else
+          @stream.puts result
```

Comment:
We should attempt to pretty print json here too probably. For example of the plan returns a Hash we converting it to json and pretty printing is probably better than just puts.

Reply:
Implemented that in the case of a Hash.